deeplinkdispatch/src/main/java/com/airbnb/deeplinkdispatch/DeepLinkEntry.java

```diff
@@ -104,7 +104,7 @@ private static String parsePath(DeepLinkUri parsedUri) {
 
   boolean matches(String inputUri) {
     DeepLinkUri deepLinkUri = DeepLinkUri.parse(inputUri);
```

Comment:
so it's completely valid that the `parse` method returns null? should it be annotated with nullable? As far as the source of that crash, should we care about why the input uri is not valid?